pages/common/pssh.md

```diff
@@ -1,27 +1,27 @@
 # pssh
 
-> Parallel ssh program.
+> Parallel SSH program.
 
-- Run a command on host1 and host2 to print the date on each server inline:
+- Run a command on two hosts, and print its output on each server inline:
 
-`pssh -i -H "{{host1}} {{host2}}" {{date}}`
+`pssh -i -H "{{host1}} {{host2}}" {{echo "foo"}}`
```

Comment:
The quotes won't be transferred through, right?

Reply:
Not sure what you mean, please expand a bit.

Reply:
Ah, oops! What I meant by this was that the quotes around `foo` won't be read by `pssh` without escaping them first, right?

Reply:
Oh, so if an echo command with a multi-word (or variable-containing) parameter is sent in this manner, the hosts won't actually see the quotes, you mean? I guess we can use something else then, for simplicity's sake. How about `ls -l`?

Reply:
Yeah, that's it - `ls -l` would work better

Reply:
What was the problem with `date` ? It's a simple command. Generates a single line. Very good for examples like these.

Reply:
It actually threw me off for a bit, as I didn't immediately realize it that parameter was meant to be an arbitrary command. My reasoning was that something like `echo`, `grep` or `ls` would make this point clearer.

Reply:
Interesting, it was pretty clear to me that this was an arbitrary command. Maybe my familiarity with the command biased me. 

Anyway, I still think `date` is the right command in this scenario, because if you have hundreds of servers, `ls -l` of the home directory will typically give you the same result. However, `date` will give you the current local time of that server. It's just a one line output and you also get to know what timezones your servers belong to ;)

Reply:
How about something like `hostname -i`, then? I included the `-i` option to give the visual hint that this isn't just a token word but an actual command.

Reply:
Sure.